### DISCUSSION TITLE
Unable to return pan.quark.cn video response data

### DISCUSSION THREAD

**Discussion by w2ch3n** [mitmproxy/mitmproxy#7396] [Intercept Everything!]
#### Problem Description
When I use mitmproxy11 to open pan.quark.cn to play a video file, it never returns the video response data.

#### Steps to reproduce the behavior:
1. To open [夸克网盘](https://pan.quark.cn/list#/list/all) you will need an account
2. and then store a video file to your web drive
3. open it up and you will find that the video keeps loading
![image](https://github.com/user-attachments/assets/bb79b167-ada5-4566-b879-f7944fd71e8a)

#### System Information
Mitmproxy: 11.0.0
Python:    3.10.11
OpenSSL:   OpenSSL 3.3.1 4 Jun 2024
Platform:  Windows-10-10.0.19045-SP0

**Comment by Prinzhorn** [comment]
You probably need the [`stream_large_bodies`](https://docs.mitmproxy.org/stable/concepts-options/#stream_large_bodies) option
